privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -502,14 +508,15 @@ def getUserList(self, searchDict):
         # do the filter depending on the searchDict
         filter = u"(&" + self.searchfilter
         for search_key in searchDict.keys():
+            # convert to unicode
+            searchDict[search_key] = to_unicode(searchDict[search_key])
```

Comment:
Basically you are right. Please advise further.
getUserList is *only* used here:
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/lib/user.py#L637
The *serachDict* is constructed in this function ``get_user_list`` and never used afterwards.